app/src/main/java/com/example/isioyemohammed/gitlist/GitListAdapter.java

```diff
@@ -121,7 +109,7 @@ public ViewHolder(View itemView) {
          */
         public void setValues(@NonNull GithubUsers userList) {
             this.textViewUsername.setText(userList.getLogin());
-            this.textViewCompany.setText(userList.getUrl());
+//            this.textViewCompany.setText(userList.getUrl());
```

Comment:
@codestaintin 
It's not good practise to leave commented out code in source control. If you aren't using this you should delete it.